addons/virtualcamera/VCameraBrain.gd

```diff
@@ -12,7 +12,7 @@ func get_highest_priority_vcamera() -> VCamera:
 	var highest_priority = 0 if cam == null else cam.priority
 	var vcams = get_tree().get_nodes_in_group("vcamera")
 	for vcam in vcams:
-		if vcam is VCamera and (cam == null or vcam.priority > highest_priority):
+		if vcam is VCamera and vcam.enabled and (cam == null or vcam.priority > highest_priority):
```

Comment:
A potential bug that I'd like to confirm this avoids would be setting `last_active_vcamera.enabled` to `false`. From what I can tell, that wouldn't actually disable it until another vcamera became active. This could be solved by setting line 11 to
```
var cam = last_active_camera if last_active_camera.enabled else null
```

Reply:
Good catch, thanks for the fix!